pkg/latency/edge/derpmap/derp_regions.go

```diff
@@ -0,0 +1,37 @@
+package derpmap
+
+// maps the DERP "RegionName" to the nearest AWS region name.
+// ref. https://aws.amazon.com/about-aws/global-infrastructure/regions_az/
+var derpRegionNameToAWSRegion = map[string]string{
+	"Amsterdam":     "eu-west-1",
+	"Bangalore":     "ap-south-1",
+	"Chicago":       "us-east-2",
+	"Dallas":        "us-west-2",
```

Comment:
is it?

Reply:
> You can enable the new Local Zone in Dallas, us-east-1-dfw-2a, from the [Zones](https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#Settings:tab=zones) tab

https://aws.amazon.com/about-aws/whats-new/2023/11/aws-new-local-zone-dallas-texas/

So, should be us-east-1, closest AWS "region"